kairon/actions/definitions/custom_parallel_actions.py

```
@@ -0,0 +1,132 @@
+        Fetches the action configuration from the database.
+
+        @return: Action configuration dictionary.
+        """
+        try:
+            p_action_config_dict = ParallelActionConfig .objects().get(
+                bot=self.bot, name=self.name, status=True
+            ).to_mongo().to_dict()
+            return p_action_config_dict
+        except DoesNotExist as e:
+            logger.exception(e)
+            raise ActionFailure("No Async Callback action found for given action and bot")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)